servers/curated-corpus-api/schema-admin.graphql

```diff
@@ -121,6 +121,16 @@ enum SectionItemRemovalReason {
   OTHER
 }
 
+"""
+Status of a Section
```

Comment:
nit: update comment: "Represents the current Section status, computed dynamically."